.github/workflows/archery.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         working-directory: dev/archery
         run: pytest -v archery
       - name: Archery Docker Validation
-        run: archery docker
+        run: archery docker validate-config
```

Comment:
Click hasn't been executing the command initialization with a subcommand, effectively omitting the validation.

Although it have not caused any problems since the docker build and run commands verify the configuration.

Reply:
Should rename to check-config similarly to the crossbow config check command.